### PR TITLE
Improve CI config

### PR DESCRIPTION
**Pull request by szepeviktor**

### CODE REVIEW

**Comment by calebporzio** [comment]
Please provide a description for why this is an improvement.

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -9,10 +9,11 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x]
+        node-version: ['12.x']
 
     steps:
-    - uses: actions/checkout@v1
+    - name: Checkout code
```
Comment (szepeviktor) [inline .github/workflows/run-tests.yml:15]: Giving code checkout a name

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -9,10 +9,11 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x]
+        node-version: ['12.x']
```
Comment (szepeviktor) [inline .github/workflows/run-tests.yml:12]: Escaping string in YAML

**Comment by szepeviktor** [comment]
@calebporzio I'm very sad that is was not possible to contribute 😿

**Pull request closed without merging**